backend/routes.js

```diff
@@ -287,6 +355,91 @@ route.GET('/file/{hash}', {
   return h.response(blobOrString).etag(hash)
 })
 
+route.POST('/deleteFile/{hash}', {
```

Comment:
Can you add a comment above this line something to the effect of, "allow file deletion, and allow either the bearer of the deletion token or the file owner to delete it" ?